src/Chapter1.hs

```diff
@@ -619,9 +631,11 @@ Implement a function that returns the sum of the last two digits of a number.
 Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
-
-sumLast2 n = error "sumLast2: Not implemented!"
-
+sumLast2 :: Int -> Int
+sumLast2 n =
+    let (withoutLast, last) = abs n `divMod` 10
+        secondToLast = withoutLast `mod` 10
```

Comment:
Great usage of standard functions 👌🏼 

You can notice that actually second last could be written with the `lastDigit` functions from the previous task. But as you correctly noticed, `abs` id not required in here, so this is already a great solution 👍🏼